rest/route/project.go

```diff
@@ -273,6 +273,12 @@ func (h *projectIDPatchHandler) Run(ctx context.Context) gimlet.Responder {
 			Message:    err.Error(),
 		})
 	}
+
+	before, err := h.sc.GetProjectSettingsEvent(identifier, dbProjectRef)
```

Comment:
should we add a test that changes get logged?

Reply:
@ybrill Can you give me an idea in which ways we can test whether changes get logged? The return value for `GetProjectSettingsEvent` is passed to `LogProjectModified` which is tested here: https://github.com/evergreen-ci/evergreen/blob/master/model/project_event_test.go#L67